.changeset/lucky-badgers-own.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core-components': minor
+'@backstage/theme': minor
+---
+
+Update default color of DismissableBanner link and button and make button color configurable
```

Comment:
Can you include an example here, please? The changesets get turned into release note so having an example will help people as they upgrade